src/playercards/PlayerCardPanelApi.ttslua

```diff
@@ -0,0 +1,49 @@
+do
+  local PlayerCardPanelApi = {}
+  local guidReferenceApi = require("core/GUIDReferenceApi")
+
+  local function getPlayerCardPanel()
+    return guidReferenceApi.getObjectByOwnerAndType("Mythos", "PlayerCardPanel")
+  end
+
+  -- Will delete all laid out cards for the specified section of the table
+  ---@param targetState The section of the table to delete cards from
+  PlayerCardPanelApi.deleteAll = function(targetState)
+    local targetState = targetState or "default"
+    getPlayerCardPanel().call("swapState", targetState)
+    return getPlayerCardPanel().call("deleteAll")
+  end
+
+  -- Spawn a given list of cards, but seperated into groups based on their card type (asset, event, or skill)
+  ---@param cardlist List containing sublists for assets, events, and skills (and optionally limited versions of each), each sublist containing a list of card ids of cards to spawn
+  ---@param name Uniquely identifying name for the group of cards to be spawned
+  ---@param startPos String for the desired start position, based on those available in the PlayerCardPanel
+  ---@param rotation The rotation for the cards to be spawned
+  ---@param spread Boolean of whether to spread out the cards (true) or stack them in a deck (false)
+  ---@param spreadCols Int of how many cards should be placed before starting a new row
+  ---@param targetState String of what section of the table to spawn the cards, based on the option in PlayerCardPanel
+  PlayerCardPanelApi.spawnCardsByType = function(params)
```

Comment:
Can have the parameters as actual separate arguments and then forward them as table like other Apis do